ci-operator/config/open-cluster-management/thanos/open-cluster-management-thanos-release-2.3.yaml

```diff
@@ -0,0 +1,45 @@
+base_images:
+  base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+  open-cluster-management_builder_go1.15-linux-amd64:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.15-linux-amd64
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.15-linux-amd64
+images:
+- dockerfile_path: Dockerfile.multi-stage
+  from: base
+  inputs:
+    open-cluster-management_builder_go1.15-linux-amd64:
+      as:
+      - registry.ci.openshift.org/open-cluster-management/builder:go1.15-linux-amd64
+  to: thanos
+promotion:
+  name: "2.3"
+  namespace: open-cluster-management
+resources:
+  '*':
+    limits:
+      memory: 6Gi
+    requests:
+      cpu: 100m
+      memory: 1Gi
+tag_specification:
+  name: "4.7"
+  namespace: ocp
+test_binary_build_commands: "true"
+tests:
+- as: e2e-aws
+  steps:
+    cluster_profile: aws
+    workflow: openshift-e2e-aws
```

Comment:
Actually, what's the point of having a presubmit that 1) installs a vanilla OCP and 2) runs shared OCP tests againts the installed cluster. None of your code is tested by this, why have it?

/hold

Reply:
Thank you for pointing it out. yes, I should remove it.

Reply:
hi @petr-muller , I have updated PR, pls help to review it. thanks.